### PR TITLE
Do not touch VNC arguments when enabling or disabling it.

### PR DESCRIPTION
**Pull request by mchf**
Updated version of: https://github.com/yast/yast-network/pull/394

### CODE REVIEW

**Comment by mchf** [comment]
Note: I've manually tested the patch and so far I was able to do vnc installation and establish vnc connection right after the installation.

**Comment by mchf** [comment]
LGTM, however I've asked for another review too since I got involved in this patch.

File: package/yast2-network.changes [hunk 1]
```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Tue Apr 19 08:01:16 UTC 2016 - mfilka@suse.com
+
+- bnc#966694
+  - Do not touch VNC arguments when enabling or disabling it.
```
Comment (mvidner) [inline package/yast2-network.changes]: Thanks! I would mention the symptom that we are fixing:
> bnc#966694: TigerVNC Viewer Error: No matching security types
 Do not touch...

**Comment by mvidner** [comment]
LGTM

File: package/yast2-network.changes (hunk 1 above)
Comment (mchf) [inline package/yast2-network.changes]: ok

**Comment by mchf** [comment]
@michalsrb thanks for patch

**Pull request merged**